content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -155,6 +155,20 @@ This label has been deprecated. Please use `kubernetes.io/arch` instead.
 
 This label has been deprecated. Please use `kubernetes.io/os` instead.
 
+### kube-aggregator.kubernetes.io/automanaged {kube-aggregator-kubernetesio-automanaged}
+
+Example: `kube-aggregator.kubernetes.io/automanaged: "manageContinuously"`
+
+Used on: APIService
+
+It is a label attached to the APIService that identifies how the APIService wants to be synced. `manageOnStart` is a value for the AutoRegisterManaged label that indicates the APIService wants to be synced one time when the controller starts.
+
+`manageOnStart = "onstart"`
+
+`manageContinuously` is a value for the AutoRegisterManaged label that indicates the APIService wants to be synced continuously.
+
+`manageContinuously = "true"`
```

Comment:
This looks like a direct quote from the code. I'd prefer to rephrase it so it makes sense to someone who doesn't understand even a symbol of Go.